src/app/_services/fullscreen.service.ts

```diff
@@ -0,0 +1,36 @@
+import { Injectable } from "@angular/core"
+
+@Injectable()
+export class FullScreenService {
```

Comment:
So I am not super familiar with Angular services, but I noticed a list of services we have listed here: https://github.com/ithaka/aiw-ui/blob/6002815ddf31785510b58039906c3a3e18a9ecd1/src/app/app.module.ts#L167

Does this new service need to be listed there?